### ISSUE TITLE
Discrepancy between Main Results and Ablation Study results

### ISSUE THREAD

**Issue by eco-deagle** [valeoai/DrivoR#41]
<img width="555" height="736" alt="Image" src="https://github.com/user-attachments/assets/3c830f89-8122-4082-94c4-a4c8e408a0a7" />
<img width="566" height="93" alt="Image" src="https://github.com/user-attachments/assets/9a13d15b-8198-4257-9daf-6830761c5362" />

The main result shows that drivoR reached 93.1 training on trainset while in the ablation study the PDMS score was 90 for drivoR. 

I am a confused by this difference. Could you please clarify if the version in the ablation study uses a compact configuration compared to the one in the main results? Or was it evaluated under a different protocol?

Thank you.

**Comment by EllingtonKirby** [comment]
Hello,
yes, we used a more compact configuration for the ablation results:

>  The model is trained on the navtrain split for 10 epochs, with learning rate 2 × 10−4 and cosine annealing scheduling. For the ablation studies, if not described otherwise, we train our model with this common base architecture. All ablation model scores are computed on the navval split.

and the final training setup which gets us the final scores was explained in section 4.2.4 in the paper. Basically we take our best performing model from the ablations and train for longer using a simple data augmentation by adding an additional time step to the ground truth trajectories.

**Comment by eco-deagle** [comment]
> Hello, yes, we used a more compact configuration for the ablation results:
> 
> > The model is trained on the navtrain split for 10 epochs, with learning rate 2 × 10−4 and cosine annealing scheduling. For the ablation studies, if not described otherwise, we train our model with this common base architecture. All ablation model scores are computed on the navval split.
> 
> and the final training setup which gets us the final scores was explained in section 4.2.4 in the paper. Basically we take our best performing model from the ablations and train for longer using a simple data augmentation by adding an additional time step to the ground truth trajectories.

Thank you very much for the detailed clarification! 
These details are very helpful for my understanding and ongoing reproduction efforts. I have no further questions for now.
